docs/getting-started/supported-languages-and-tools.md

```diff
@@ -391,9 +391,9 @@ The table below lists all languages and frameworks that Codacy supports and the
 <sup><span id="swiftlint-complexity">5</span></sup>: Supports [reporting warnings or errors](https://realm.github.io/SwiftLint/cyclomatic_complexity.html) on functions above specific complexity thresholds. Enable the rule **Cyclomatic Complexity** on the [Code patterns page](../repositories-configure/configuring-code-patterns.md), or use a [configuration file](https://realm.github.io/SwiftLint/index.html#configuration) to customize the thresholds.  
 <sup><span id="suggest-fixes">🔧</span></sup>: Supports [suggesting fixes](../repositories-configure/integrations/github-integration.md#suggest-fixes) for identified issues.
 
-## Codacy wrappers for supported tools
+## Docker images of supported tools
 
-Codacy adds support for new languages and tools by using ["wrapper" Docker images for running each tool](https://github.com/codacy/codacy-example-tool).
+Codacy adds support for new languages and tools by using [a Docker image for each tool](https://github.com/codacy/codacy-example-tool).
```

Comment:
@prcr @claudiacarpinteiro 

I removed any mention of wrappers and chose to refer directly to Docker images in the title to make this searchable. I also updated the body copy to match.

Let me know!

Reply:
Looks good! 👍

Reply:
The verb "run" used before helped me understand slightly better how the Docker images related to the supported tools.

What do you think about this?

> Codacy adds support for new languages and tools by using a Docker image **to run** each tool

Reply:
Perfect! Tweaked as suggested.